spinn_front_end_common/interface/interface_functions/routing_setup.py

```diff
@@ -32,50 +31,36 @@ def routing_setup() -> None:
         This does not load any routes into them.
     """
     transceiver = FecDataView.get_transceiver()
-    routing_tables = FecDataView.get_uncompressed().routing_tables
-    progress = ProgressBar(len(routing_tables), "Preparing Routing Tables")
-
-    # Clear the routing table for each table that needs to be set up
-    # and set up the diagnostics
-    for table in progress.over(routing_tables):
```

Comment:
As the iteration has been removed I suggest moving what little is left here to the Transceiver(s) and have ASB call the Transceiver method directly